src/ensembl/src/content/app/browser/browserSelectors.ts

```diff
@@ -71,15 +71,23 @@ export const getDefaultChrLocation = (state: RootState) => {
   }
   const { chromosome, start, end } = activeEnsObject.location;
 
-  return [chromosome, start, end] as ChrLocation;
+  return [chromosome, start, end];
 };
 
-export const getBrowserMessageCount = (state: RootState): number => {
-  return state.browser.browserEntity.messageCounter;
-};
+export const getRegionEditorActive = (state: RootState) =>
+  state.browser.browserLocation.regionEditorActive;
+
+export const getRegionFieldActive = (state: RootState) =>
+  state.browser.browserLocation.regionFieldActive;
+
+export const getRegionValidationInfo = (state: RootState) =>
+  state.browser.browserLocation.regionValidationInfo;
```

Comment:
I know we had this conversation, but here's an example, where I would prefer the selector to be named `isRegionEditorActive`, and the redux state field to be named the same way as well. `getRegionEditorActive` is very stilted (what am I getting?) :-(

Don't insist on this change if you are opposed to it; just indicating my preference.